webapp/src/client/client.ts

```diff
@@ -24,7 +19,7 @@ export default class Client {
     }
 
     setUserStatus = async (userId: string | null, status: string) => {
-        return this.doPut(`${this.baseUrl}/api/v4/users/${userId}/status`, {user_id: userId, status});
+        return this.doPut(`${this.url}/api/v4/users/${userId}/status`, {user_id: userId, status});
```

Comment:
Is there any Client4 function in the redux repo that do this for you?

Reply:
Right, we should definitely use the `Client4` function if it's available

Reply:
nice catch @larkox , will fix it. There is one from redux selectors but it expects diff. set of parameters and has no explanation.
@mickmister Earlier I imported client4 directly, hence changed it to this.(inspired from plugin-zoom)

Reply:
`mattermost-redux/src/client/client4.ts` has a function `updateStatus` that could be used here. You can directly import Client4 from `mattermost-redux/src/client`. It should be properly configured.

Reply:
@larkox you mean we should use `updateStatus` and delete function `setUserStatus`? won't it be out of scope for this PR?

Reply:
OK. Let's leave it like this for now, and we will open a new ticket.